Sources/App/Views/DocumentationPageProcessor.swift

```diff
@@ -235,4 +258,19 @@ struct DocumentationPageProcessor {
     func relativeDocumentationURL(reference: String, docArchive: String) -> String {
         "/\(repositoryOwner)/\(repositoryName)/\(reference)/documentation/\(docArchive.lowercased())"
     }
+
+    var latestStableLinkExplanatoryText: String {
+        switch referenceKind {
+            case .release: return "This is documentation from an old version of \(packageName)."
+            case .preRelease: return "This documentation is from a pre-release of \(packageName) and may not reflect the latest stable release."
+            case .defaultBranch: return "This documentation is from the \(packageName) \(reference) branch and may not reflect the latest stable release."
+            default: return ""
+        }
+    }
+}
+
+extension Array where Element == DocumentationPageProcessor.AvailableDocumentationVersion {
+    var latestStableVersion: DocumentationPageProcessor.AvailableDocumentationVersion? {
+        self.filter(\.isLatestStable).first
```

Comment:
```suggestion
        first(where: \.isLatestStable)
```

but tbh I wouldn't push a simple filter into an extension like this. I think this is clearer at the call site as `first(where: \.isLatestStable)` because it tells you how it's finding that version.

Reply:
Yea, when I first made the extension, it did more than this but eventually came down to a filter on a boolean. Good catch 👍